README.md

```diff
@@ -55,10 +55,13 @@ The file format used is [HOCON](https://github.com/lightbend/config/blob/master/
 To see how all this works in action, head over to the [Getting Started](docs/GettingStarted.md) page.
 
 # Major Contributors
-At the moment, there is one contributors which has done major investments into this project:
+At the moment we have the following contributors on this project:
```

Comment:
Can we delete this line at all? In my opinion the caption says enough.

Reply:
Good point, done: https://github.com/smart-data-lake/smart-data-lake/pull/1/commits/34f03bd35bcb030f01e0fc0b98831a0f1a2a3432